app/domain/authentication/authn_jwt/signing_key/create_signing_key_provider.rb

```diff
@@ -2,7 +2,7 @@ module Authentication
   module AuthnJwt
     module SigningKey
       # Factory that returns the interface implementation of FetchSigningKey
-      CreateSigningKeyFactory ||= CommandClass.new(
+      CreateSigningKeyProvider ||= CommandClass.new(
```

Comment:
can you change the method `create_signing_key` to `create_signing_key_provider`?